xml/System.Collections/Hashtable.xml

```diff
@@ -98,6 +98,7 @@
  [!code-cpp[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/cpp/hashtable_example.cpp#00)]
  [!code-csharp[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/cs/hashtable_example.cs#00)]
  [!code-vb[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/vb/hashtable_example.vb#00)]  
+ [!code-powershell[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/powershell/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/ps/hashtable_example.ps1#00]
```

Comment:
This is not rendering correctly somehow:
![image](https://user-images.githubusercontent.com/12971179/32853086-07b6c58e-c9ef-11e7-81bf-2bf33acd902b.png)

Not sure if it's an error in our tooling our what. Can you try to remove the #00 from the end for me to try to rule out if tags are not working for PowerShell samples?